app/views/summaries/_summary.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="page-section">
+  <h3>Checkouts: <%= patron.checkouts.length %></h3>
+
+  <% count = patron.checkouts.select(&:recalled?).length %>
```

Comment:
I wonder why this is in a view and not the controller (or a helper) 🤔

Reply:
hmm yes let's move it when we are working on the views